notebooks/xvcfmerge-array-input.py

```diff
@@ -0,0 +1,126 @@
+     - [xvcfmerge](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfmerge:xbrianh-input-format?tab=info)
+
+    Workflow execution time is typically ~20 minutes per VCF.
+    """
+
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with callysto.Cell("python"):
+    import os
+    from firecloud import fiss
+
+    # Function to upload TSVs to a Terra Data Table
+    def upload_data_table(tsv: str):
+        resp = fiss.fapi.upload_entities(os.environ['GOOGLE_PROJECT'],
+                                         os.environ['WORKSPACE_NAME'],
+                                         tsv,
+                                         model="flexible")
+        resp.raise_for_status()
+
+    # Function to modify Terra Data Table rows
+    def update_row(table: str, row_name: str, updates: dict):
+        fiss_updates = [fiss.fapi._attr_set(column, value)
+                        for column, value in updates.items()]
+        resp = fiss.fapi.update_entity(os.environ['GOOGLE_PROJECT'],
+                                       os.environ['WORKSPACE_NAME'],
+                                       table,
+                                       row_name,
+                                       fiss_updates)
+        resp.raise_for_status()
+
+with callysto.Cell("python"):
+    # Prepare the merge workflow input data table for DRS URIs.
+    # This is a typical workflow preparation for merging TOPMed VCFs _without_ downloading them to your workspace bucket.
+    # Results will be placed in your workspace bucket.
+    bucket = os.environ['WORKSPACE_BUCKET']
+    table = "vcf-merge-workflow-input-drs"
+
+    tsv_data = "\t".join([f"{table}_id", "workspace", "billing_project"])
+    for row_name in ["drs_combined_a", "drs_combined_b"]:
+        tsv_data += os.linesep + "\t".join([row_name, os.environ['WORKSPACE_NAME'], os.environ['GOOGLE_PROJECT']])
+    upload_data_table(tsv_data)
+
+    update_row(table, row_name="drs_combined_a", updates=dict(inputs=["drs://dg.4503/697f611b-aa8a-4bd7-a80b-946276273833",
+                                                                      "drs://dg.4503/ce212b62-e796-4b32-becb-361f272cead0"],
+                                                              output=f"{bucket}/merged/drs_combined_a.vcf.gz"))
+
+    update_row(table, row_name="drs_combined_b", updates=dict(inputs=["drs://dg.4503/93286e47-3d09-47e6-ac87-4c2975ef0c3f",
+                                                                      "drs://dg.4503/aba6b011-2ab4-4739-beb4-c1eeaee60c74"],
+                                                              output=f"{bucket}/merged/drs_combined_b.vcf.gz"))
+
+with callysto.Cell("python"):
+    # Prepare the merge workflow input data table for Google Storage URIs.
+    # This workflow preparation uses VCFs that are present in your workspace bucket.
```

Comment:
or with DRS links?

Reply:
There example preparation in the previous cell works with DRS links.